core/src/main/java/org/kiwix/kiwixmobile/core/settings/SliderPreference.java

```diff
@@ -171,9 +170,12 @@ private void setGrayState(View view) {
 
     TextView titleView = view.findViewById(android.R.id.title);
     TextView summaryTV = view.findViewById(android.R.id.summary);
-    if (!enabled) {
-      titleView.setTextColor(Color.GRAY);
-      summaryTV.setTextColor(Color.GRAY);
+    if (enabled) {
```

Comment:
just delete these, this whole method should be deleted. Setting the enabled state of the parent view should mean that state gets propagated downwards to its children and this already occurs in `CorePrefsFragment`
```
  private void setSliderState() {
    boolean enabled = getPreferenceManager().getSharedPreferences().getBoolean(
      PREF_ZOOM_ENABLED, false);
    if (enabled) {
      mSlider.setEnabled(true);
    } else {
      mSlider.setEnabled(false);
    }
  }
```
albeit in a very longwinded way could be 1 line
```
mSlider.setEnabled(sharedPrefUtil.getPrefZoomEnabled());
```

Reply:
> just delete these, this whole method should be deleted. Setting the enabled state of the parent view should mean that state gets propagated downwards to its children and this already occurs in `CorePrefsFragment`
> 
> ```
>   private void setSliderState() {
>     boolean enabled = getPreferenceManager().getSharedPreferences().getBoolean(
>       PREF_ZOOM_ENABLED, false);
>     if (enabled) {
>       mSlider.setEnabled(true);
>     } else {
>       mSlider.setEnabled(false);
>     }
>   }
> ```
> 
> albeit in a very longwinded way could be 1 line
> 
> ```
> mSlider.setEnabled(sharedPrefUtil.getPrefZoomEnabled());
> ```

@macgills This has been implemented in the latest commit.
Please review!

Reply:
All working great, thanks for the work